### PR TITLE
fix: bug fixed for lse loans when including eco friendly and single p…

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- LSE loans bug fixed when including eco friendly and single parent tags

### CODE REVIEW

File: @kiva/kv-components/vue/KvClassicLoanCard.vue [hunk 1]
```diff
@@ -453,12 +457,12 @@ export default {
 
 			if (!this.categoryPageName) {
 				if (categories.ecoFriendly
-					&& !callouts.filter((c) => c.toUpperCase() === categories.ecoFriendly.toUpperCase()).length) {
+					&& !callouts.find((c) => c === ECO_FRIENDLY_KEY || c === SUSTAINABLE_AG_KEY)) {
```
Comment (dyersituations) [inline @kiva/kv-components/vue/KvClassicLoanCard.vue:460]: Do we still need case insensitive checking here?

Comment (roger-in-kiva) [reply @kiva/kv-components/vue/KvClassicLoanCard.vue]: Looks like test is passing, but good for consistency. ty

**Comment by dyersituations** [comment]
The changes don't look that different from what was there previously. What specifically caused the issue?

**Comment by roger-in-kiva** [comment]
> The changes don't look that different from what was there previously. What specifically caused the issue?

Previously we were comparing with this `categories.ecoFriendly.toUpperCase()` but `categories.ecoFriendly` is boolean.
Now we are checking if keys exist in the callouts array.

**Review by dyersituations — APPROVED** [review]

**Pull request merged**